docs/user-guide/lazy/query_plan.md

```diff
@@ -11,17 +11,16 @@ We can understand both the non-optimized and optimized query plans with visualiz
 
 ### Graphviz visualization
 
-First we visualise the non-optimized plan by setting `optimized=False`.
-
-{{code_block('user-guide/lazy/query_plan','plan',['show_graph'])}}
-
 <div style="display:none">
 ```python exec="on" result="text" session="user-guide/lazy/query_plan"
 --8<-- "python/user-guide/lazy/query_plan.py:setup"
---8<-- "python/user-guide/lazy/query_plan.py:plan"
```

Comment:
Removing this will cause errors later on as the variable q1 is not defined anymore for the query_plan.py:createplan snippet

Reply:
Sorry, should be fixed now. I've also rebased the branch on top of the latest `master`.